modules/tobira/src/main/java/org/opencastproject/tobira/impl/TobiraEndpoint.java

```diff
@@ -204,4 +219,35 @@ private static Response badRequest(String msg) {
     logger.warn("Bad request to tobira/harvest: {}", msg);
     return Response.status(BAD_REQUEST).entity(msg).build();
   }
+
+  @POST
+  @Path("/stats")
+  @Consumes(APPLICATION_JSON)
+  @RestQuery(
+      name = "stats",
+      description = "Accepts a json blob of statistical data about Tobira.",
+      restParameters = {},
+      bodyParameter = @RestParameter(description = "The job blob",
+                                     isRequired = true,
+                                     name = "BODY",
+                                     type = Type.STRING),
```

Comment:
I think the latest style guide would say this (same format as in your prior commits).

```suggestion
      bodyParameter = @RestParameter(
            description = "The job blob",
            isRequired = true,
            name = "BODY",
            type = Type.STRING
      ),
```

Also, "the job blob" is likely a copy and paste error?

Reply:
Yep